base/simplebook/ansible/roles/simplebook/tasks/main.yml

```diff
@@ -5,6 +5,7 @@
     name: "{{ item }}"
     state: present
   loop:
+    - nodejs
```

Comment:
node is installed later directly (the debian nodejs isn't necessarily the one we want / we want to lock in the node version installed) -- https://github.com/OpenTechStrategies/torque-sites/blob/c2009892b0e129005711c6088b18a6641f0eeadd/base/simplebook/ansible/roles/simplebook/tasks/main.yml#L70-L87

at the very least I don't know if it makes sense to install it both ways.